__tests__/app.test.js

```diff
@@ -72,6 +71,26 @@ describe('GET /api/articles', () => {
         })
     })
 })
+describe('get /api/articles/:article_id', ()=> {
+    test('request returns article object with the specified id ', ()=> {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(response => {
+            const article = response.body.articles[0]
```

Comment:
You're only expecting one article back from this endpoint, so the array is unnecessary. You can return the element [0] from earlier on so that all you're expecting here is the one article not in an object.